crates/runtime/src/trampolines/aarch64.rs

```diff
@@ -0,0 +1,119 @@
+use wasmtime_asm_macros::asm_func;
+
+#[rustfmt::skip]
+asm_func!(
+    "host_to_wasm_trampoline",
+    r#"
+        .cfi_startproc
+        bti c
+
+        // Load the pointer to `VMRuntimeLimits` in `x9`.
+        ldur x9, [x1, #8]
+
+        // Check to see if callee is a core `VMContext` (MAGIC == "core").
+        ldur w10, [x0]
+        mov  w11, #0x6f63
```

Comment:
Maybe add a comment stating that we do not support big-endian AArch64 (also on line 46)?